cmssw-tool-conf.spec

```diff
@@ -119,6 +119,7 @@ Requires: fftjet-toolfile
 Requires: lapack-toolfile
 Requires: pyminuit2-toolfile
 Requires: professor-toolfile
+Requires: professor2-toolfile
```

Comment:
professor2 is very different from professor. It does not yet have all functionality of professor. This is the reason to keep the old version. Note that CMSSW code is not compiled against this external.